### ISSUE TITLE
XLib warning printed to terminal on application close

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fyne#1160]
### Describe the bug:
It looks like https://github.com/fyne-io/fyne/pull/1153 accidentally introduced a weird Xlib error on application close. I don't know why I didn't notice it when testing the PR though. Seems to just be a warning and not something that break anything as of now, however it would be good to get rid of it :)

### To Reproduce:
Steps to reproduce the behaviour:
1. Use an Nvidia GPU on Linux and close the application.

### Screenshots:
```bash
Xlib:  extension "NV-GLX" missing on display ":0".
```
### Example code:
It works fine to test this using `fyne_demo`.

### Device (please complete the following information):
 - OS: Solus Budgie (Linux distro with desktop using Mutter WM and X11 session)
 - Version: Linux kernel 5.6.18
 - Go version: 1.14.3
 - Fyne version: 15dd658e6eaac9c9564be2c9590e7d7a132a148d

**Comment by andydotxyz** [comment]
Is this still present on 1.3.2?

**Comment by Jacalz** [comment]
Yes I vanställ troligare this on the latest tag in the develop branch.

**Comment by andydotxyz** [comment]
Is it at all like this issue? https://askubuntu.com/questions/67567/how-to-fix-error-xlib-extension-nv-glx-missing-on-display

**Comment by Jacalz** [comment]
Nope. This is a desktop computer. I guess I could technically could force the intel iGPU to be turned off in the bios but I very much doubt that it would make a difference as I am using the Nvidia GPU and nothing else.

**Comment by andydotxyz** [comment]
Sorry I wasn't suggesting tweaking the hardware, was just trying to figure some more out.
Which graphics driver is X using?

**Comment by Jacalz** [comment]
I didn’t think you did. I just added my thoughts and I guess that they came out a bit wrong 🙃

It should be using the proprietary Nvidia driver 450 or possibly 380, but I will need to double check that when I have time 😅

**Comment by barjoco** [comment]
I'm also getting `Xlib:  extension "NV-GLX" missing on display ":0"` on close. I'm also on Solus. Is this a Solus-specific thing? Any fix for it?

**Comment by AlbinoGeek** [comment]
This is present on Fedora Workstation 33 running Gnome Shell 3.38.2 and nvidia 455 driver via dnf
`xorg-x11-drv-nvidia.x86_64                 3:455.45.01-1.fc33 @rpmfusion-nonfree-updates`

**Comment by GeekStocks** [comment]
Confirming this is still present on Ubuntu 20.04 LTS (linux/amd64) running Gnome 3.36.8 and nVidia driver 390. Go version 1.16.5 and Fyne version 2.0.3

Exact error text is: Xlib:  extension "NV-GLX" missing on display ":1"

Everything looks fine in the app (graphics-wise). The warning on os.exit is just unsightly is all. Hope that helps.

**Comment by justinfx** [comment]
I'm seeing this as well on Ubuntu 18.04

**Comment by paboum** [comment]
Also present for `:0` on gentoo with `nvidia-drivers-515.65.01`.

**Comment by jljlspeedrunning** [comment]
It looks like this is the same problem for Manjaro (with X11 and Nvidia GPU).

**Comment by andydotxyz** [comment]
Does the system also have an integrated GPU?

**Comment by Jacalz** [comment]
I can't speak for anyone else here but FYI: When I initially saw this, it was on a desktop computer without an integrated GPU :)
